### DISCUSSION TITLE
aspire hosting qdrant  - shows connection "other" when referenced from other projects

### DISCUSSION THREAD

**Discussion by markat1** [microsoft/aspire#6077] [Q&A] [answered]
I'm referencing aspire hosting qdrant on my main python project, but it doesn't seem like it maps connection for qdrant correctly.

Anyone knows why connection says "other" for aspire qdrant connection?

![image](https://github.com/user-attachments/assets/f4d198ce-c081-4dc2-8369-eb2379452bba)

here's how I build it:

![image](https://github.com/user-attachments/assets/c66cae63-c47e-4f17-9b6f-51b4b6c504cd)

on my main project I reference the vectordb

![image](https://github.com/user-attachments/assets/879f35f1-f059-448f-9373-740d83aac48b)

**Comment by davidfowl** [comment, marked as answer]
It's because secrets don't show up in the aspire dashboard in azure.

**Reply by markat1** [reply]
@davidfowl oh okay I see! how would I access these secrets?
